module/VuFind/src/VuFind/Content/Reviews/PluginManager.php

```diff
@@ -57,7 +57,6 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'demo' => Demo::class,
         'guardian' => Guardian::class,
         'syndetics' => Syndetics::class,
-        'syndeticsplus' => SyndeticsPlus::class,
```

Comment:
...and here, though in this case there's already a Deprecated handler we could use if that's preferable to pointing syndeticsplus at syndetics.